README.md

```diff
@@ -29,75 +35,20 @@ Instead, create your own new node project, and install the oss-mariner package v
 ### Step-by-step
 
 1. Create a new project folder and use `npm init` to make it a node project.
-1. Copy the contents of `runFasterCode.ts` into `index.js` in the new project.
-   1.1. <https://github.com/indeedeng/Mariner/blob/master/examples/runFasterCode.ts>
+1. Copy the contents of `runFasterCode.ts` into `index.js` and copy `config.json`, `exampleData.json`
+   in the new project. - <https://github.com/indeedeng/Mariner/blob/master/examples/runFasterCode.ts> - <https://github.com/indeedeng/Mariner/blob/master/examples/config.json> - <https://github.com/indeedeng/Mariner/blob/master/examples/exampleData.json>
```

Comment:
I think the links should be sub-bullets. right now they just have dashes between them, which looks odd.

Reply:
yeah thats strange, I just fixed that